python/amici/pysb_import.py

```diff
@@ -1114,9 +1116,9 @@ def has_fixed_parameter_ic(specie: pysb.core.ComplexPattern,
         ODE model
 
     :return:
-        False if the species does not have an initial condition at all.
+        ``false`` if the species does not have an initial condition at all.
```

Comment:
```suggestion
        ``False`` if the species does not have an initial condition at all.
```

Reply:
Although I don't like sentences starting in lower-case, I am not sure if one should change case of symbols.

Reply:
False is upper case in python ... or am I messing up with R here ...

Reply:
No... I'm messing up with C++